Jellyfin.Windows.Tray/TrayApplicationContext.cs

```diff
@@ -106,24 +109,41 @@ private void CreateTrayIcon()
             };
         }
 
-        private void LoadPortFromJellyfinConfig()
+        private void LoadJellyfinConfig()
         {
                 RegistryKey registryKey = Registry.LocalMachine.OpenSubKey("Software\\WOW6432Node\\Jellyfin\\Server");
                 _installFolder = registryKey.GetValue("InstallFolder").ToString();
                 _dataFolder = registryKey.GetValue("DataFolder").ToString();
                 _configFile = Path.Combine(_dataFolder, "config\\system.xml").ToString();
+                _networkFile = Path.Combine(_dataFolder, "config\\network.xml").ToString();
                 _executableFile = Path.Combine(_installFolder, "jellyfin.exe");
+                _port = "8096";
 
                 if (File.Exists(_configFile))
                 {
                     XDocument systemXml = XDocument.Load(_configFile);
                     XPathNavigator SettingsReader = systemXml.CreateNavigator();
 
                     FirstRunDone = SettingsReader.SelectSingleNode("/ServerConfiguration/IsStartupWizardCompleted").ValueAsBoolean;
-                    string port = SettingsReader.SelectSingleNode("/ServerConfiguration/PublicPort").Value;
+                    var publicPort = SettingsReader.SelectSingleNode("/ServerConfiguration/PublicPort")?.Value;
+                    if (!string.IsNullOrEmpty(publicPort))
+                    {
+                        _port = publicPort;
+                    }
```

Comment:
Should also attempt to pull this from network.xml also?

Reply:
Yeah, it should. This PR is big enough, I will add it in the next bit.